main.go

```diff
@@ -20,3 +22,19 @@ func helloHandler(w http.ResponseWriter, r *http.Request) {
 	args := mux.Vars(r)
 	fmt.Fprintf(w, "Hello %s!", args["name"])
 }
+
+func jobHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can you add documentation to this function. [Here](https://golang.org/doc/effective_go.html#commentary) are some tips for writing Go documentation.
> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

Also, can we name it something slightly more semantic? How about `initiateTranscriptionJobHandler`? Or is that too verbose?